recipe/meta.yaml

```diff
@@ -75,7 +73,7 @@ requirements:
     - libnetcdf * {{ mpi_prefix }}_*  # [tempest == 'tempest']
     - netcdf-cxx-legacy  # [tempest == 'tempest']
```

Comment:
Btw, neither Tempest, nor MOAB needs the NetCDF-cxx-legacy libraries anymore.

Reply:
But this version does, right?

Reply:
By "anymore" do you mean in the upcoming release?

Reply:
Yes the last official release of Tempest does. But I wanted to point out that when we have a new release of Tempest, that works with MOAB release, we won't need this extra dependency anymore.

Reply:
Yes upcoming release; or current Tempest master

Reply:
Would now be a good time to start at PR where we test-build the current master?  Or should I still wait a bit?

Reply:
The MOAB PR is not merged yet. There are other things enhancement requests in that branch that are currently delaying the review. But I will see if I can cherry-pick and submit a new PR to just get MOAB master to build cleanly with Tempest master.

So short answer, wait until next Monday.

Reply:
No rush, just asking.  I started #29 as a test bed but that uses tempest-remap 2.0.3.  If we want to use their master, we can't do that on conda-forge itself, it would have to be tested locally by someone.  Much trickier!